keychain/src/lib.rs

```diff
@@ -19,6 +19,8 @@ extern crate byteorder;
 extern crate grin_util as util;
 extern crate rand;
 extern crate serde;
+// #[allow(unused_imports)]
+#[cfg(test)]
```

Comment:
Nice.

Reply:
I think that commenting it out means it's ignored. On the master branch I made it a bit more clear, but since TravisCI already passed and the testnet1 branch is on a withering milestone I got lazy :)